android/filament-utils-android/src/main/cpp/Utils.cpp

```diff
@@ -73,6 +73,13 @@ static jlong nCreateSkybox(JNIEnv* env, jclass,
     return (jlong) Skybox::Builder().environment(cubemap).showSun(true).build(*engine);
 }
 
+static void nGetSphericalHarmonics(JNIEnv* env, jclass, jobject javaBuffer, jint remaining, jobject outBuffer) {
+    AutoBuffer buffer(env, javaBuffer, remaining);
+    KtxBundle bundle((const uint8_t*) buffer.getData(), buffer.getSize());
+    AutoBuffer outAutoBuffer(env, outBuffer, 3 * 9, true);
+    bundle.getSphericalHarmonics((float3*) outAutoBuffer.getData());
```

Comment:
What happens btw if the KTX doesn't contain spherical harmonics here?

Reply:
Now it would return null. Btw, `nCreateIndirectLight` also does no checks.